napari/_qt/layers/qt_base_layer.py

```diff
@@ -47,62 +55,40 @@ def __init__(self, layer):
         sld.setMinimum(0)
         sld.setMaximum(100)
         sld.setSingleStep(1)
-        sld.valueChanged.connect(self.changeOpacity)
+
+        self.emit_opacity_event = lambda value: self.events.opacity(
+            value=value / 100
+        )
```

Comment:
i think we can make the slider go from 0 to 1 (which is how we like to think of opacity) in steps of 0.01 rather than 0 to 100 in steps of 1, then this won't be needed. Similarly in other places where the variable is transformed for use with a slider

Reply:
Yea it seemed to be just these two sliders so if we could change the ranges that'd be great

Reply:
@tlambert03 or I should be able to do that quickly in a stand alone PR

Reply:
@sofroniewn did you ever get this in? I couldn't find such a PR...